legal-api/src/legal_api/services/authz.py

```diff
@@ -615,6 +620,9 @@ def business_blocker_check(business: Business, is_ignore_draft_blockers: bool =
     if not business.good_standing:
         business_blocker_checks[BusinessBlocker.NOT_IN_GOOD_STANDING] = True
 
+    if business.is_pending_amalgamating_business:
```

Comment:
Your tests fail because `is_pending_amalgamating_business` isn't an instance method.

Should be something like `if Business.is_pending_amalgamating_business(business.identifier):`

Reply:
Thank you. This fixed it. Working on adding unit tests now